site/src/components/radio.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Radio
+description: The radio component in the Atlas Design System
+template: standard
+figmaEmbed: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FuVA2amRR71yJZ0GS6RI6zL%2F%25F0%259F%258C%259E-Atlas-Design-Library%3Fnode-id%3D838%253A1096"
+---
+
+# Radio
+
+A radio button allows users to select a single option. It is usually associated with form submissions.
+
+<div class="table-wrapper margin-top-xs">
```

Comment:
I think we should be making documentation tables in markdown

Reply:
Updated